english/quick_draw/AI_Pictionary.md

```diff
@@ -0,0 +1,5 @@
+It is a lot of fun playing pictionary with a bunch of friends. 
+
+But have you ever played pictionary with a computer, where you draw sketches and computer predicts your sketch within 20 seconds?
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.